src/aks-preview/azcli_aks_live_test/scripts/test_cli_unit.sh

```diff
@@ -9,6 +9,8 @@ set -o xtrace
 # check var
 [[ -z "${WIKI_LINK}" ]] && (echo "WIKI_LINK is empty"; exit 1)
 [[ -z "${ACS_BASE_DIR}" ]] && (echo "ACS_BASE_DIR is empty"; exit 1)
+[[ -z "${IGNORE_EXIT_CODE}" ]] && (echo "IGNORE_EXIT_CODE is empty")
```

Comment:
curious where is this used? in which case we ignore exit code.

Reply:
Actually this is not used by the check-in pipeline, we always exit with the error code.

This is used in our runner, even some of the test cases may fail, the status of the pod/container after execution should be completed.